engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableAjTest.java

```diff
@@ -524,6 +524,11 @@ public void testAjRandomStaticOverflow() {
 
     @SuppressWarnings("SameParameterValue")
     private void testAjRandomStaticOverflow(int seed, int leftSize, int rightSize) {
+        if (AsOfJoinHelper.USE_TYPED_STATE_MANAGER) {
```

Comment:
aj() will need to rehash; though we should not use a load factor greater than 0.75.  It is probably the case that we will want this test to cause rehashing if we do not have another suitable one.

Reply:
restored this as it properly tests the OA rehashing while building